content/docs/learning-guide/tutorials/assets/mesh-assets.md

```diff
@@ -2,69 +2,64 @@
+    In the preceding image, there is a single **Mesh group**. By default, all the meshes in a source asset are processed as a single mesh group. Each mesh group produces a set of product assets. You can create additional mesh groups by choosing **Add another mesh**.
 
-    For this tutorial, you can use the default mesh group with all meshes selected.
+    The **Name mesh** property contains the name of the source asset. All product assets of this mesh group use this string as a prefix for their name. When there are multiple mesh groups, the product assets appear in Asset Browser beneath the source asset and are organized by mesh group name.
 
-1. The customizations you make in Scene Settings are stored in a *sidecar file* with a `.assetinfo` extension. When Asset Processor detects a `.assetinfo` file, it uses the settings in the file to process the related source asset. This sidecar file is treated as a source dependency for the asset. This means that if the `.assetinfo` file is changed, the source asset will be reprocessed even if the source asset has not changed.
+    The **Select meshes** property reads **All meshes selected**. You can choose the {{< icon browse-edit-select-files.svg >}} **node select** button to select which meshes to include in the mesh group.
 
-    Let's add a modifier to customize how the asset is processed. Choose the **Add Modifier** button to view the mesh modifier list and select **Coordinate system change**.
+1. Add a modifier to customize how the asset is processed. Choose the **Add Modifier** button to view the mesh modifier list and select [**Coordinate system change**](/docs/user-guide/assets/scene-settings/meshes-tab#coordinate-system-change).
 
     ![ Scene Settings meshes tab, adding mesh a modifier. ](/images/learning-guide/tutorials/assets/meshes-coordinate-system-change.png)
 
 1. The Coordinate system change mesh modifier is used to scale or transform the asset for those scenarios where the asset might be too small, too large, or incorrectly oriented in O3DE. By default, the modifier provides a single option to rotate the mesh 180 degrees. Activate the **Use advanced settings** toggle to expose the advanced modifier settings.
 
     ![ Scene Settings meshes Coordinate system change modifier, Use advanced settings. ](/images/learning-guide/tutorials/assets/meshes-use-advanced-settings.png)
 
-1. Let's customize the scale of the asset. Set the **Scale** property to `5.0` to scale the asset to five times its size.
-
-    {{< note >}}
-You can learn about the advanced settings properties of the Coordinate system change modifier, as well as the other available modifiers and their properties, in the [Scene Settings](/docs/user-guide/assets/scene-settings) topic of the **User Guide**.
-    {{< /note >}}
+1. Customize the scale of the asset. Set the **Scale** property to `5.0` to scale the asset to five times its size.

```

Comment:
There is no `Scale` property in the **Meshes** tab or in any of its modifiers. Maybe it was an old property that now has been removed.

Reply:
It's in the advanced settings of the Coordinate System Change modifier enabled in the previous step. These steps are simply to introduce users to modifiers and modifying modifier properties.

Reply:
I see... I missed that. Thanks for clarifying!